File: java/ng/src/main/java/com/betfair/aping/enums/OrderBy.java
package com.betfair.aping.enums;

public enum OrderBy {
    /**
     * Use BY_PLACE_TIME instead.
     */
    @Deprecated
    BY_BET,
    BY_MARKET,
    BY_MATCH_TIME,
    BY_PLACE_TIME,
    BY_SETTLED_TIME,
    BY_VOID_TIME;
}

File: cSharp/Api-ng-sample-code/Api-ng-sample-code/TO/OrderBy.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace Api_ng_sample_code.TO
{
    [JsonConverter(typeof(StringEnumConverter))]
    public enum OrderBy
    {
        BY_BET, 
        BY_MARKET,
        BY_MATCH_TIME, 
        BY_PLACE_TIME, 
        BY_SETTLED_TIME, 
        BY_VOID_TIME
    }
}
